parser-typechecker/src/Unison/Codebase/Editor/HandleInput.hs

```diff
@@ -283,7 +283,7 @@ loop = do
             ParseErrors text [ err | Result.Parsing err <- toList notes ]
           Just (Left errNames) -> do
             ns <- makeShadowedPrintNamesFromHQ hqs errNames
-            ppe <- prettyPrintEnv (Names3.suffixify ns)
+            ppe <- suffixifiedPPE ns
```

Comment:
Rather than creating a `PrettyPrintEnv` from a suffixified `Names`, you instead can just directly create a suffixified `PrettyPrintEnv` from a regular `Names`.